pom.xml

```diff
@@ -57,7 +57,7 @@
     <properties>
         <ivy.plugin.version>1.21</ivy.plugin.version>
         <mockito.version>1.10.19</mockito.version>
-        <jenkins.version>1.625.3</jenkins.version>
+        <jenkins.version>2.14</jenkins.version>
```

Comment:
It won't work properly. JCasC requires 2.60.3 at least. It is fine to bump the dependency (and to set java.level to 8)